R/module_nested_tabs.R

```diff
@@ -103,7 +115,17 @@ ui_teal_module.teal_module <- function(id, modules, depth = 0L) {
       if (depth >= 2L) tags$div(style = "mt-6"),
       if (!is.null(modules$datanames)) {
         fluidRow(
-          column(width = 9, ui_teal, class = "teal_primary_col"),
+          column(
+            width = 9,
+            div(
+              div(
+                class = "teal_validated"
+                # todo: should error message be displayed here?
+              ),
+              ui_teal
+            ),
+            class = "teal_primary_col"
+          ),
```

Comment:
revert the change. nothing is needed here. this div was extended to include `ui_validate_teal_data` that no longer exists
https://github.com/insightsengineering/teal/pull/1330/files#diff-f7278c16079bf8eeee343aba99b73c110c99d6cc705b7bcdbcc0df3c8bde35abR112
```suggestion
          column(width = 9, ui_teal, class = "teal_primary_col"),
```